lib/logger/lib/logger.ex

```diff
@@ -470,14 +472,14 @@ defmodule Logger do
 
   """
   @spec compare_levels(level, level) :: :lt | :eq | :gt
-  def compare_levels(level, level), do:
-    :eq
-  def compare_levels(left, right), do:
-    if(level_to_number(left) > level_to_number(right), do: :gt, else: :lt)
+  def compare_levels(level, level), do: :eq
+
+  def compare_levels(left, right),
+    do: if(level_to_number(left) > level_to_number(right), do: :gt, else: :lt)
```

Comment:
Is this okay or shouldn't it be inline in the first place?

```elixir
def compare_levels(left, right) do
  if(level_to_number(left) > level_to_number(right), do: :gt, else: :lt)
end
```

Reply:
Let's drop the `do:` syntax IMO and have

```elixir
def compare_levels(level, level) do
  :eq
end

def compare_levels(left, right) do
  if level_to_number(left) > level_to_number(right), do: :gt, else: :lt
end
```